### DISCUSSION TITLE
[Prettier Plugin] Enabling in liquid files

### DISCUSSION THREAD

**Discussion by lukecharle** [tailwindlabs/tailwindcss#9664] [Help]
Hi, it seems Prettier is disabled in my liquid file (see screenshot), which means I'm then not able to actually use the tailwind plugin to prettify my markup. I don't want to have to install the shopify liquid prettier extension though (Using VS code). Any ideas?

<img width="516" alt="Screenshot 2022-10-26 at 11 31 55" src="https://user-images.githubusercontent.com/115987937/198004178-cfef81e7-03dc-4a00-86ed-ba6cca8ca8d3.png">

**Comment by lspoor** [comment]
Anybody know anything about this?

**Comment by cannap** [comment]
are you tried https://marketplace.visualstudio.com/items?itemName=heybourn.headwind  

but when you click on the prettier icon what is the output?
